sdk/eventhub/event-hubs/src/util/connectionStringUtils.ts

```diff
@@ -9,8 +9,8 @@ import { parseConnectionString } from "@azure/core-amqp";
 export interface EventHubConnectionStringProperties {
   /**
    * The fully qualified Event Hub namespace extracted from the "Endpoint" in the
-   * connection string. This is likely to be similar to "{yournamespace}.servicebus.windows.net".
-   * This is typically used to construct the EventHub{Producer|Consumer}Client.
+   * connection string. This is likely to be similar to "\{yournamespace\}.servicebus.windows.net".
```

Comment:
@deyaaeldeen Is this the recommended way to get past the error from tsdoc linter on malformed tags?

![image](https://user-images.githubusercontent.com/16890566/107682310-05303f00-6c55-11eb-8371-19195445c053.png)

Reply:
I do not remember if I have seen this before but I think if you enclose it in `` instead of "" the error will go away. If not, perhaps try <> instead of {}.

Reply:
Ah, backtick works! Updated in https://github.com/Azure/azure-sdk-for-js/pull/13746/commits/e97da9efdf7007d4532f68d85c8fc90307b2cdd5